streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsMetadataStateTest.java

```diff
@@ -122,28 +120,32 @@ public Object apply(final Object value) {
 
         cluster = new Cluster(null, Collections.<Node>emptyList(), partitionInfos, Collections.<String>emptySet(), Collections.<String>emptySet());
         metadataState = new StreamsMetadataState(TopologyWrapper.getInternalTopologyBuilder(builder.build()), hostOne);
-        metadataState.onChange(hostToPartitions, cluster);
-        partitioner = new StreamPartitioner<String, Object>() {
-            @Override
-            public Integer partition(final String topic, final String key, final Object value, final int numPartitions) {
-                return 1;
-            }
-        };
+        metadataState.onChange(hostToActivePartitions, hostToStandbyPartitions, cluster);
+        partitioner = (topic, key, value, numPartitions) -> 1;
     }
 
     @Test
-    public void shouldNotThrowNPEWhenOnChangeNotCalled() {
+    public void shouldNotThrowExceptionWhenOnChangeNotCalled() {
```

Comment:
This is fine, but it seems like we should also make some assertions about what gets returned in this case.

Reply:
This is just a small rename, while I was there, since the test name was checking for NPE, when it would fail if there was any exception..